src/upload/system/library/transbank/utils/TransbankSdkWebpay.php

```diff
@@ -39,15 +40,11 @@ public function initTransaction($amount, $sessionId, $buyOrder, $returnUrl)
                     "url" => $response->url,
                     "token_ws" => $response->token
                 ];
-            } else {
-                throw new \Exception('No se ha creado la transacción para, amount: ' . $amount . ', sessionId: ' . $sessionId . ', buyOrder: ' . $buyOrder);
             }
-        } catch (\Exception $e) {
-            $result = [
-                "error" => 'Error al crear la transacción',
-                "detail" => $e->getMessage()
-            ];
-            $this->log->logError(json_encode($result));
+        }  catch (TransactionCreateException $e) {
+
+            $this->log->logError(json_encode($e->getMessage()));
```

Comment:
`json_encode()` is necessary? `$e->getMessage()` already return a string

Reply:
It is not necessary, the corresponding modifications will be made